src/components/slack/attachment/attachment.component.ts

```diff
@@ -11,6 +11,7 @@ import { SettingService } from '../../../services/setting.service';
 export class SlackAttachmentComponent {
     @Input() attachment: Attachment;
     @Input() parser: SlackParser;
```

Comment:
SlackAttachmentComponent.parser is no longer used anywhere because it is completely replaced by SlackAttachmentComponent.attachmentTextParser, am I right?

If this is true, please remove attachmentTextParser.parser from attachment.component.ts.